pkgs/data/fonts/last-resort/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, fetchurl }:
+
+let
+  version = "13.001";
+in fetchurl {
+  name = "last-resort-${version}";
```

Comment:
I wonder if we should maybe call this package `last-resort-font` (or its attribute name?). Upstream uses it for the repo at least and it might make it more discoverable.

Reply:
So, I pondered this: the repo is 'last-resort-font', but the font itself is called 'Last Resort'. nixpkgs doesn't seem to have a consistent pattern (in fact, if anything, it's the opposite: there are a lot of packages that add '-font' to the name attribute, and several where the name ends in '-font' but the attribute doesn't). Additionally, the paths within nixpkgs are sometimes inconsistent (e.g. `tewi-font` is in `pkgs/data/fonts/tewi`).

I'm not opposed to changing this, but do you have any kind of consistent pattern to propose?

Reply:
The attr, pname and directory should in most cases match each other except if a suffix/prefix was added for a variant or there is another compelling reason to name it differently. For example the Prometheus exporters are all named in the same pattern despite some of them having a different name in the sources/repositories.

Reply:
> I'm not opposed to changing this, but do you have any kind of consistent pattern to propose?

No, it's fine either way, I'd say.